subscription-service/src/main/kotlin/com/egm/stellio/subscription/service/SubscriptionService.kt

```diff
@@ -523,4 +525,8 @@ class SubscriptionService(
     private var rowToSubscriptionCount: ((Row) -> Int) = { row ->
         row.get("count", Integer::class.java)!!.toInt()
     }
+
+    private var rowToSub: (Row) -> String = { row ->
+        row.get("sub", String::class.java)!!
```

Comment:
It can't be null ? What if it is null, shouldn't we map to an error known in our project ?

Reply:
it can't be null, the sub is a mandatory field.